app/controllers/actions/CheckIsCurrentReturnPeriodFilter.scala

```diff
@@ -83,6 +83,8 @@ class CheckIsCurrentReturnPeriodFilterImpl(startReturnPeriod: Period,
             Redirect(controllers.previousReturns.routes.SubmittedReturnForPeriodController.onPageLoad(EmptyWaypoints, startReturnPeriod))
           case Excluded =>
             Redirect(controllers.routes.CannotStartExcludedReturnController.onPageLoad())
+          case Expired =>
```

Comment:
if there are two cases that match you can use an or operator `case Excluded | Expired`